index.js

```diff
@@ -34,24 +38,27 @@ module.exports = class Tailor extends EventEmitter {
             }
             return new Buffer(`${memoizedDefinition}var ${pipeInstanceName}=${PIPE_DEFINITION}</script>\n`);
         };
+
         const requestOptions = Object.assign({
             fetchContext: () => Promise.resolve({}),
             fetchTemplate: fetchTemplate(
-                options.templatesPath ||
+                templatesPath ||
                 path.join(process.cwd(), 'templates')
             ),
             fragmentTag: 'fragment',
             handledTags: [],
             handleTag: () => '',
             requestFragment,
-            pipeDefinition: (pipeInstanceName) => pipeChunk(amdLoaderUrl, pipeInstanceName),
+            pipeDefinition: pipeInstanceName => pipeChunk(amdLoaderUrl, pipeInstanceName),
```

Comment:
Yeah. We can curry a new function since `pipeInstanceName` is anyway the second param